src/benchmarks/micro/libraries/System.Runtime/Perf.String.cs

```diff
@@ -154,13 +154,20 @@ public string TrimEnd_CharArr(string s, char[] c)
             => s.TrimEnd(c);
 
         [Benchmark]
-        [Arguments("Hello", 'l', '!')] // Contains two 'l'
-        [Arguments("Hello", 'a', 'b')] // Contains one 'a'
-        [Arguments("This is a very nice sentence", 'z', 'y')] // 'z' does not exist in the string
-        [Arguments("This is a very nice sentence", 'i', 'I')] // 'i' occuress 3 times in the string
+        [ArgumentsSource(nameof(ReplaceArguments))]
         public string Replace_Char(string text, char oldChar, char newChar)
             => text.Replace(oldChar, newChar);
 
+        public static IEnumerable<object[]> ReplaceArguments()
+        {
+            yield return new object[] { "Hello", 'l', '!' };    // Contains two 'l'
+            yield return new object[] { "Hello", 'a', 'b' };    // Contains one 'a'
+            yield return new object[] { "This is a very nice sentence", 'z', 'y' }; // 'z' does not exist in the string
+            yield return new object[] { "This is a very nice sentence", 'i', 'I' }; // 'i' occuress 3 times in the string
```

Comment:
```suggestion
            yield return new object[] { "This is a very nice sentence", 'i', 'I' }; // 'i' occurs 3 times in the string
```

Reply:
Haha, copy & pasted the type too. Thanks!